providers/common/provider_seeding.c

```diff
@@ -18,26 +18,30 @@ static OSSL_FUNC_cleanup_nonce_fn *c_cleanup_nonce = NULL;
 int ossl_prov_seeding_from_dispatch(const OSSL_DISPATCH *fns)
 {
     for (; fns->function_id != 0; fns++) {
+        /*
+        * We do not support the scenario of an application linked against
+        * multiple versions of libcrypto (e.g. one static and one dynamic), but
+        * sharing a single fips.so. We do a simple sanity check here.
```

Comment:
This is common for all providers so perhaps `sharing a single provider.so.`

Reply:
Also the indentation is off by one space.

Reply:
Actually it only impacts the fips provider. The built-in providers (default, base, null) will always be associated with the version of libcryto they are built into. So it should never impact those. The legacy provider is linked against libcrypto and doesn't use any up-calls, so it doesn't impact that one either. That only leaves fips.so as being affected by this.

Reply:
Ah, ok, so only the indentation... :grin: